_releaser/Dockerfile

```diff
@@ -42,13 +42,13 @@ FROM base AS aws-cloudfront-update
 ARG AWS_REGION
 ARG AWS_LAMBDA_FUNCTION
 ARG AWS_CLOUDFRONT_ID
+ENV AWS_LAMBDA_FUNCTION_FILE="cloudfront-lambda-redirects.js"
+ENV REDIRECTS_FILE="/site/redirects.json"
+ENV REDIRECTS_PREFIXES_FILE="redirects-prefixes.json"
```

Comment:
Not really important (mostly pondering) I guess these could be `ARG` if they're only used at build-time (or do we run the image that's produced?)